src/test/java/com/cloudant/tests/CloudantClientTestBase.java

```diff
@@ -0,0 +1,82 @@
+package com.cloudant.tests;
+
+import com.cloudant.client.api.CloudantClient;
+
+import org.junit.After;
+import org.junit.Before;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Created by Rhys Short on 21/05/15.
+ */
+public abstract class CloudantClientTestBase {
```

Comment:
well since all classes need to make Cloudant client for the test to start, I'd argue that to remove repetition and make it less painful if we change the Contructor for CloudantClient, but I'm happy to go with the consensus thoughts @brynh ?